docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -1,22 +1,23 @@
 # Install OpenVINO™ Runtime on macOS from an Archive File {#openvino_docs_install_guides_installing_openvino_from_archive_macos}
 
-With the OpenVINO™ 2022.2 release, you can download and use archive files to install OpenVINO Runtime. The archive files contain pre-built binaries and library files needed for OpenVINO Runtime, as well as code samples.
+With the OpenVINO™ 2022.3 release, you can download and use archive files to install OpenVINO Runtime. The archive files contain pre-built binaries and library files needed for OpenVINO Runtime, as well as code samples.
 
 Installing OpenVINO Runtime from archive files is recommended for C++ developers. If you are working with Python, the PyPI package has everything needed for Python development and deployment on CPU and GPUs. Visit the [Install OpenVINO from PyPI](installing-openvino-pip.md) page for instructions on how to install OpenVINO Runtime for Python using PyPI.
 
 See the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes) for more information on updates in the latest release.
 
 > **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter can be installed via [pypi.org](https://pypi.org/project/openvino-dev/) only.
 
-> **NOTE**: The Intel® Distribution of OpenVINO™ toolkit is supported on macOS version 10.15 with Intel® processor-based machines.
+> **NOTE**: Intel® Distribution of OpenVINO™ toolkit is supported on macOS with Intel® processor-based machines.
 
 ## System Requirements
 
 @sphinxdirective
 
 .. tab:: Operating Systems
 
-  macOS 10.15, 11, 12, and 13 versions with the x86 architecture, 64 bits
+   * macOS 10.15, 11, 12, and 13, x86, 64-bit
+   * macOS 10.15, 11, 12, and 13, ARM, 64-bit
   
   .. note::
    
```

Comment:
Question regarding the following note:
```
Only macOS 10.15 is fully validated with OpenVINO. Other versions depend on the compatibility of macOS.
```
Are macOS versions 11, 12, 13 are validated and missed in this note?
If not - do we officially support macOS 11, 12 and 13 without validation?

Reply:
Yes, that was suggested by Helena and Ilya Lavrenov, macOS 11, 12 and 13 are supported but not validated.